src/strings/PowerFxResources.en-US.resx

```diff
@@ -6615,4 +6615,8 @@
     <value>This feature is deprecated and is no longer supported.</value>
     <comment>An error message for deprecated features.</comment>
   </data>
+  <data name="ErrExpectedIdentifierArg_Name" xml:space="preserve">
```

Comment:
We have a loc team. Only change en-US, the others will get overwritten

Reply:
We need to provide strings for other languages as we'd get a "Resource not found" exception if we don't provide those, when this message will be used.
I've checked with the localization team and they will ignore the other resource strings, so this is fine to keep those.

Reply:
If that's happening, we have a separate bug. We should be falling back to en-us if the string isn't found.